src/promptflow/promptflow/_constants.py

```diff
@@ -45,6 +45,13 @@ class FlowLanguage:
     CSharp = "csharp"
 
 
+class FlowType:
+    """The enum of flow type."""
+
+    YAML_FLOW = "dag"
```

Comment:
DAG_FLOW?

Reply:
Regarding this name, I prefer to use **YAML_FLOW** and **EAGER_FLOW** because in other parts of our code, we use **eager_flow** and **yaml_flow**. 
If we change it to **dag_flow** and **flex_flow**, it can easily cause confusion and look like a new concept different from eager and yaml.

But we can recording as dag and flex in our Telemetry first.
![image](https://github.com/microsoft/promptflow/assets/23182548/400faa46-f011-4eba-bb0a-d4ba6ce7da85)

Reply:
@crazygao is working on the eager -> flex renaming thing. It's better to change to the new name when we introduce new code. Btw, are there somewhere we referenced DAG flow as YAML flow

Reply:
ok, I have already modified it.